gqlalchemy/utilities.py

```diff
@@ -41,14 +59,16 @@ def to_cypher_value(value: Any) -> str:
     return f"'{value}'"
 
 
-def to_cypher_properties(properties: Optional[Dict[str, Any]] = None) -> str:
+def to_cypher_properties(
+    properties: Optional[Dict[str, Any]] = None, config: NetworkXCypherConfig = NetworkXCypherConfig()
```

Comment:
I would remove the default construction of `NetworkXCypherConfig` from all of these functions in utilities